environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/freeipa/FreeIpaCreationHandler.java

```diff
@@ -367,6 +370,19 @@ private EnvCreationEvent getNextStepObject(EnvironmentDto environmentDto) {
                 .build();
     }
 
+    private void setImage(EnvironmentDto environment, CreateFreeIpaRequest createFreeIpaRequest) {
+        if (environment.getFreeIpaCreation() != null) {
+            String imageCatalog = environment.getFreeIpaCreation().getImageCatalog();
```

Comment:
idea: the catalog and the id should be in a Image object

Reply:
I'd leave it as it is. This structure is already the part of the freeipa api.

Reply:
check DistroXImageV1Request we should be more consistent I think. So in this case because this is a new API I suggest to use such what we use for Distrox

Reply:
fixed